swing/src/net/sf/openrocket/gui/dialogs/ComponentAnalysisDialog.java

```diff
@@ -42,18 +42,14 @@
 import net.sf.openrocket.aerodynamics.AerodynamicCalculator;
 import net.sf.openrocket.aerodynamics.AerodynamicForces;
 import net.sf.openrocket.aerodynamics.FlightConditions;
+import net.sf.openrocket.gui.components.*;
```

Comment:
Wildcard imports are not very efficient. I think IntelliJ does this automatically, so this is probably not your fault, but please write explicit imports.